actionsystemtest/lib/action_system_test/test_helpers/assertions.rb

```diff
@@ -0,0 +1,33 @@
+module ActionSystemTest
+  module TestHelpers
+    # Assertions for system testing that aren't included by default in Capybara.
+    # These are assertions that are useful specifically for Rails applications.
+    module Assertions
+      # Asserts that all of the provided selectors are present on the given page.
+      # Using the selector, +:avatar+ for example, asserts that the selector is
+      # present with the items passed in. If options are provided, the assertion
+      # will check that each item is present with those options as well.
+      #
+      #   assert_all_of_selectors(:avatar, 'Eileen', 'Jeremy')
+      #   assert_all_of_selectors(:avatar, 'Eileen', 'Jeremy', visible: all)
+      def assert_all_of_selectors(selector, *items, **options)
```

Comment:
I've created a PR for adding these methods to Capybara - https://github.com/rails/rails/pull/26703#issuecomment-269984808 - the only real difference is that the :wait option will apply to the group as a whole rather than each individual selector check